cpu/stm32f0/periph/gpio.c

```diff
@@ -341,209 +152,77 @@ int gpio_init_int(gpio_t dev, gpio_pp_t pullup, gpio_flank_t flank, gpio_cb_t cb
     return 0;
 }
 
-void gpio_irq_enable(gpio_t dev)
+void gpio_init_af(gpio_t pin, gpio_af_t af)
```

Comment:
the compiler says: 'gpio_af_t` undefined.

You need to add a typedef for this to `cpu/stm32f0/include/periph_cpu.h`. It needs to look like this:
```c


/**
 * @brief   Available MUX values for configuring a pin's alternate function
 */
typedef enum {
    GPIO_AF0 = 0,           /**< use alternate function 0 */
    GPIO_AF1,               /**< use alternate function 1 */
    GPIO_AF2,               /**< use alternate function 2 */
    GPIO_AF3,               /**< use alternate function 3 */
    GPIO_AF4,               /**< use alternate function 4 */
    GPIO_AF5,               /**< use alternate function 5 */
    GPIO_AF6,               /**< use alternate function 6 */
    GPIO_AF7,               /**< use alternate function 7 */
} gpio_af_t;
```